ui/page/debug_page.go

```diff
@@ -162,7 +162,18 @@ func (pg *DebugPage) Layout(gtx C) D {
 		return sp.Layout(pg.ParentWindow(), gtx)
 
 	}
-	return components.UniformPadding(gtx, container)
+	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
+		return pg.layoutMobile(gtx, container)
+	}
+	return pg.layoutDesktop(gtx, container)
+}
+
+func (pg *DebugPage) layoutDesktop(gtx layout.Context, body layout.Widget) layout.Dimensions {
+	return components.UniformPadding(gtx, body)
+}
+
+func (pg *DebugPage) layoutMobile(gtx layout.Context, body layout.Widget) layout.Dimensions {
+	return components.UniformMobile(gtx, false, body)
```

Comment:
I dont see the need for the `layoutDesktop` and `layoutMobile` functions as they are only used once.

```suggestion
	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
               // mobile layout
		return components.UniformMobile(gtx, false, container)
	}
	// desktop layout
	return components.UniformPadding(gtx, container) 
}
	
```

Reply:
we should leave as is, further details here https://github.com/planetdecred/godcr/pull/971#discussion_r904149266

Reply:
Then i believe it wont be out of order to properly comments these codes for easier reviews.

Reply:
yeah, your right